Sources/TuistGenerator/Generator/ConfigGenerator.swift

```diff
@@ -191,5 +191,24 @@ final class ConfigGenerator: ConfigGenerating {
                 }
             }
         }
+
+        if let deploymentTarget = target.deploymentTarget {
+            switch deploymentTarget {
+            case let .iOS(version, devices):
+                var deviceFamilyValues: [Int] = []
+                if devices.contains(.iphone) { deviceFamilyValues.append(1) }
+                if devices.contains(.ipad) { deviceFamilyValues.append(2) }
+
+                settings["TARGETED_DEVICE_FAMILY"] = .string(deviceFamilyValues.map { "\($0)" }.joined(separator: ","))
+                settings["IPHONEOS_DEPLOYMENT_TARGET"] = .string(version)
+
+                if devices.contains(.ipad), devices.contains(.mac) {
```

Comment:
ah didn't realise this ... is the iPad checkbox a pre-requisite to the Mac checkbox ? i.e. is `[.iphone, .mac]` an invalid option?

Reply:
> is the iPad checkbox a pre-requisite to the Mac checkbox ?

Exactly 👍 

![image](https://user-images.githubusercontent.com/10795657/66892589-635efa80-efec-11e9-8f37-be2c424db901.png)

☝️ If the `iPad` is un-checked, we cannot select a `Mac`.

<img width="767" alt="Screen Shot 2019-10-15 at 21 22 51" src="https://user-images.githubusercontent.com/10795657/66892663-94d7c600-efec-11e9-969c-52b772c2f348.png">

☝️ If the `iPad` is checked, we can select a `Mac`. 

> is [.iphone, .mac] an invalid option?

Yes, that is why we need the condition I added.